docs/CONTRIBUTING.md

```diff
@@ -98,6 +98,71 @@ Updating documentation is an important part of the contribution process. If you
 
 To improve the in-editor experience when using Clerk's SDKs, we do our best to add [JSDoc comments](https://jsdoc.app/about-getting-started.html) to our package's public exports. The JSDoc comments should not attempt to duplicate any existing type information, but should provide meaningful additional context or references. If you are adding a new export, make sure it has a JSDoc comment. If you are updating an existing export, make sure any existing comments are updated appropriately.
 
+There are some styleguide decisions you should follow when authoring JSDoc comments:
```

Comment:
This list is non-exhaustive, I'm sure I'll add more over time. But that's a good start